.github/actions/goreleaser-build-sign-publish/action_utils

```diff
@@ -27,8 +28,12 @@ _publish_snapshot_manifests() {
   local docker_manifest_extra_args=$DOCKER_MANIFEST_EXTRA_ARGS
   local full_sha=$(git rev-parse HEAD)
   local images=$(docker images --filter "label=org.opencontainers.image.revision=$full_sha" --format "{{.Repository}}:{{.Tag}}" | sort)
-  local arches=(amd64 arm64)
   local raw_manifest_lists=""
+  if [[ $ENABLE_GORELEASER_SPLIT == "true" ]]; then
```

Comment:
WDYT about creating multi-platform docker images? 

https://goreleaser.com/cookbooks/multi-platform-docker-images/

Can be created as a follow-up or future work too.

Reply:
we are already using the multi-platform docker images :)

this is to split it based on the goarch passed in for build